Client/Impl/Commands/FailJobCommand.cs

```diff
@@ -53,6 +53,12 @@ public IFailJobCommandStep2 ErrorMessage(string errorMsg)
             return this;
         }
 
+        public IFailJobCommandStep2 RetryBackOff(TimeSpan retryBackOff)
+        {
+            request.RetryBackOff = (long)Math.Round(retryBackOff.TotalMilliseconds, MidpointRounding.AwayFromZero);
```

Comment:
:wrench: Is this really necessary? I would feel better reducing the complexity here, and be more consistent with the other places were we use the timespan.

Reply:
@Zelldon ok, I've fixed this